data/schema.yaml

```diff
@@ -90,14 +88,11 @@ components:
               example: http://www.bbc.co.uk/ontologies/passport/home/News
             articleType:
               type: string
-              enum: [analysis, feature]
+              enum: [analysis, feature, news]
               example: analysis
         seoHeadline:
           type: string
           example: UK pledges extra £44m for Channel border security
-        socialHeadline:
```

Comment:
Why has this been removed?

Reply:
It's still required, so needs to be added to our scenarios.

Reply:
ok

Reply:
Thanks for clarifying. @PaulChinkwende we should add this back in across the schema and I've open the issue https://github.com/bbc/simorgh/issues/262 to get these added to the scenarios.